lib/uwb/include/uwb/UwbSession.hxx

```diff
@@ -239,6 +257,24 @@ private:
     virtual std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
     GetApplicationConfigurationParametersImpl() = 0;
 
+    /**
+     * @brief Get the Application Configuration Parameters object
+     *
+     * @param requestedTypes
+     * @return std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
+     */
+    virtual std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameter>
+    GetApplicationConfigurationParametersImpl(std::vector<::uwb::protocol::fira::UwbApplicationConfigurationParameterType> requestedTypes) = 0;
```

Comment:
This seems to already exist on line 258. Can we remove one?

Reply:
updated